Dockerfile

```diff
@@ -43,6 +43,13 @@ RUN yum install -y \
         gperftools-devel && \
     yum clean all --enablerepo=* && rm -rf /var/cache/yum/
 
+# If necessary, download the authfile-update version that can do per-instance authfiles
+RUN \
+    if /usr/local/bin/pkg-cmp-lt.sh xcache 3.2.0; then \
+        curl -Lo /usr/libexec/xcache/authfile-update https://raw.githubusercontent.com/opensciencegrid/xcache/V3-branch/src/authfile-update && \
+        chmod +x /usr/libexec/xcache/authfile-update; \
```

Comment:
Just to be explicit i might suggest `chmod 755` here as the `w` bits depend on the umask when the curl command is run (you might end up with 755 or 775) ... but that's pretty minor and unlikely to cause problems either way, so up to you however you like it best.

Reply:
sure, thanks